### PR TITLE
Feedback Preview v1 [Jan29]

### PR DESCRIPTION
**Pull request by mihow**

### CODE REVIEW

**Comment by mihow** [comment]
screenshot

This is wrong, what's up Michael
<details><summary>Browser metadata</summary>





```
Path:      /projects/18
Browser:   Chrome 120.0.0.0 on Mac OS 10.15.7
Viewport:  1220 x 669 @2x
Language:  en-US
Cookies:   Disabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects%2F18&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=120.0&resolution=1220x669&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects/18) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bb0053477e290008283dac&commentID=65bbb8e07366e119bb3e9573&resolution=resolved)

**Comment by joellearob** [comment]
image. Create a page for the 'About' content instead of a popup window. 
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c00398a61dc0a8f48f46ef&resolution=resolved)

**Comment by joellearob** [comment]
Not mandatory, but you might consider adding a footer to the website with some useful information/links (for example, a contact form or email address).
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c003cbd0b1bfaa8a45a121&resolution=resolved)

**Comment by joellearob** [comment]
image  These two pages are a bit confusing from a user perspective, since they are essentially identical. Here are a few suggestions for improving the experience in that way:
Use a unique visual on each page to help differentiate them;
Sign-up page: Add information on the targeted audience:  
Who can sign-up to the platform? Is there a user validation process on your end? What features does it give the user access to?
Login page: There doesn't seem to be any way of recovering a lost/forgotten password. 

<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c00432d164b6aaf0beb0dc&resolution=resolved)

**Comment by joellearob** [comment]
image @Michael I will send you a suggestion for navigation via email. :) 
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c004d39ae486abf42a6016&resolution=resolved)

**Comment by joellearob** [comment]
image The logo is a bit too small; is there a higher quality version?
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c0051fccf320aa6edb2a8e&resolution=resolved)

**Comment by joellearob** [comment]
image Wih a few exceptions, the font size across the website is very small and difficult to read, especially in the main menu and project descriptions. The font size for body text on a website is generally between 16 and 18 px.
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c005d8e5d39fa8929ad5cf&resolution=resolved)

**Comment by joellearob** [comment]
image This depends on the logo and font sizes; however, I would suggest increasing the size of the navigation bar slightly.
<details><summary>Browser metadata</summary>





```
Path:      /projects
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fprojects&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/projects) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c0063cf55590acf448ccee&resolution=resolved)

**Comment by joellearob** [comment]
image Same comment as for the 'About' content; avoid the popup window if possible. :) 
<details><summary>Browser metadata</summary>





```
Path:      /auth/sign-up
Browser:   Chrome 119.0.0.0 on Mac OS 10.15.7
Viewport:  1440 x 657 @2x
Language:  fr-FR
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-339--ami-web.netlify.app%2Fauth%2Fsign-up&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=119.0&resolution=1440x657&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-339--ami-web.netlify.app/auth/sign-up) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65bde5aaed213100086260d8&commentID=65c00696b90990ac3d794a6d&resolution=resolved)

**Comment by mihow** [comment]
Thanks @jonbeu, will you try choosing a Data Source, Site and Device for the deployment before clicking save? We will work on highlighting the correct field that needs fixing, or at least displaying the error.

**Comment by mihow** [comment]
Hi @jonbeu, PR #366 should fix the issue you faced when creating a new deployment. The site, device & data source are no-longer required -- but also a default device & site are created for each project now. I am anxious to see some test images from the AMI-Mini! -Michael

**Comment by mihow** [comment]
@annavik There is some feedback here we should make sure to capture, then we can close this before the next feedback release. I believe I fixed all of the errors mentioned. Tessa R. left some good comments. She is familiar with another system for processing & reviewing bird audio clips (. I wonder if we can make it clear how clicking on a Taxon name does different things in different views. Tooltips may help explain Occurrences vs. Detections. I could be convinced to hide the Detections vs. Occurrences difference from the user. e.g. call everything a "Detection" but a detection could be grouped or single. However Occurrence does follow the vocabulary from the Darwincore standard https://techdocs.gbif.org/en/openapi/v1/occurrence

**Comment by mihow** [comment]
@annavik @joseepoirier The remaining TODOs for this one is to make sure we have captured all of the feedback from our user interviews into tickets or documents somewhere. @annavik can you summarize what interviews you looked through and which ones remain? Here is the summary doc:
https://docs.google.com/document/d/11pjZjSIn-sCbPZu0wi89-XmWBKDtt7B3W0ykaBK-SjQ/edit

**Comment by mihow** [comment]
Thank you everyone who participated in our first round of feedback for the preview of the AMI Platform! We recorded over 300 points of feedback! We have already fixed many of the issues raised. Others have become dedicated GitHub tickets and others are still being organized and discussed as part of larger changes. I am closing this PR and we will be putting out a new preview link soon with many of these changes and opening the platform up for additional feedback. 

I am putting our raw spreadsheet of feedback points here for reference, however it is set to private because it includes names of the contributors. 
https://docs.google.com/spreadsheets/d/1bBeHqZklbHOX9El3_JurzkSKX6Bor9h26iqArHxd2Es/edit?gid=1736625285#gid=1736625285
